kenkono/e-learning-laravel
# ELS-TASK-12-showAnswers

## Description / Purpose (checkpoints)
- Show the result of your choices.

## How To Test This PR
Go to 'localhost:8000/home' and select `select contents` then select 'learn.
Answer question and you can see that this is correct or not with color.
 
## **Commands**
Commands needed
`php artisan migrate` 
Insert the test data
`php artisan migrate:refresh --seed`
